cohortextractor/patients.py

```diff
@@ -1378,39 +1382,91 @@ def with_test_result_in_sgss(
         episode. This may change, but is set as it is due to limited
         information around re-infection and virus clearance.
 
+    # All Tests Dataset
+
+    This dataset is not subject to the same restriction as above and we expect
+    each individual test result (postive or negative) to appear in this
+    regardless of whether they are considered as within the same infection
+    episode. In an ideal world we could use just this dataset, but there are
+    some fields we need (e.g. SGTF) which are only supplied on the "earliest
+    specimen" dataset.
+
+    # S-Gene Target Failure
+
+    Using the `returning="s_gene_target_failure"` option provides additional
+    output from PCR tests results which can be used as a proxy for the presence
+    of certain Variants of Concern.
+
+    Possible values are "", "0", "1", "9"
+
+    Definitions (from email from PHE)
+
+      1: Isolate with confirmed SGTF
+      Undetectable S gene; CT value (CH3) =0
+      Detectable ORF1ab gene; CT value (CH2) <=30 and >0
+      Detectable N gene; CT value (CH1) <=30 and >0
+
+      0: S gene detected
+      Detectable S gene (CH3>0)
+      Detectable y ORF1ab CT value (CH1) <=30 and >0
+      Detectable N gene CT value (CH2) <=30 and >0
+
+      9: Cannot be classified
+
+      Null are where the target is not S Gene. I think LFTs are currently
+      also coming across as 9 so will need to review those to null as well as
+      clearly this is a PCR only variable.
```

Comment:
Does this snippet of info need actioning? Either way I'm not sure it'll be useful to end-users in our documentation as it is?